libtelemetry/src/test/java/com/mapbox/android/telemetry/MapboxTelemetryTest.java

```diff
@@ -175,6 +188,8 @@ public void checksEnabled() throws Exception {
     MapboxTelemetry theMapboxTelemetry = new MapboxTelemetry(mockedContext, aValidAccessToken, aValidUserAgent,
       mockedEventsQueue, mockedTelemetryClient, mockedHttpCallback, mockedSchedulerFlusher, mockedClock,
       mockedLocalBroadcastManager);
+    Mockito.when(MapboxTelemetry.applicationContext.getSharedPreferences(anyString(), anyInt()))
```

Comment:
This is not needed per above comment.
The same thing happens to the following tests.